C/tests/c4DatabaseTest.cc

```diff
@@ -740,8 +740,8 @@ N_WAY_TEST_CASE_METHOD(C4DatabaseTest, "Test delete while database open", "[Data
     CHECK(timeTaken < 2s);
     CHECK(err.code == kC4ErrorBusy);
 
-    auto message = slice(c4error_getDescription(err));
-    CHECK(message == "Can't delete db file while the caller has open connections");
+    string message = string(slice(c4error_getDescription(err)));
+    CHECK(message.find("Can't delete db file while the caller has open connections") != string::npos);
```

Comment:
FYI, slice has a `find` method too, so no need to convert to string

Reply:
I found an interesting thing here. Originally, Jim has it, auto message = slice(c4error_getDescription(err)). This caused sanitizer error. This is the main reason I changed it to string. Because of the time pressure, I did not spend too much to find out why.

Reply:
Oh sorry, add the `_sl` suffix to the string there and I be that will fix the sanitizer error?